batched/dense/impl/KokkosBatched_SVD_Serial_Impl.hpp

```diff
@@ -23,12 +23,14 @@
 namespace KokkosBatched {
 // Version which computes the full factorization
 template <typename AViewType, typename UViewType, typename VViewType,
-          typename SViewType, typename WViewType>
+          typename SViewType, typename WViewType, 
+          typename TolType>
```

Comment:
Do we really need a new template paramter for TolType, would it make sense to use the something like:
`Kokkos::ArithTraits<typename AViewType::non_const_value_type>::mag_type`?